### PR TITLE
Update v3 Python Worker Version to 1.1.10

### PR DESCRIPTION
**Pull request by Hazhzeng**
Python Worker Release note [1.1.10](https://github.com/Azure/azure-functions-python-worker/releases/tag/1.1.10)

Miscellaneous:
Converting CRLF => LF to make the newline character consistent.

### CODE REVIEW

File: src/Azure.Functions.Cli/StaticResources/Dockerfile.python39 [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿# To enable ssh & remote debugging on app service change the base image to the one below
-# FROM mcr.microsoft.com/azure-functions/python:3.0.15066-python3.9-appservice
-FROM mcr.microsoft.com/azure-functions/python:3.0.15066-python3.9
+# FROM mcr.microsoft.com/azure-functions/python:3.0.15149-python3.9-appservice
```
Comment (Hazhzeng) [inline src/Azure.Functions.Cli/StaticResources/Dockerfile.python39:2]: Please change this line to
```
# FROM mcr.microsoft.com/azure-functions/python:3.0-python3.9-appservice
```

File: src/Azure.Functions.Cli/StaticResources/Dockerfile.python39 [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+FROM mcr.microsoft.com/azure-functions/python:3.0.15149-python3.9
```
Comment (Hazhzeng) [inline src/Azure.Functions.Cli/StaticResources/Dockerfile.python39:3]: Please change this line to
```
# FROM mcr.microsoft.com/azure-functions/python:3.0-python3.9
```

File: src/Azure.Functions.Cli/Helpers/PythonHelpers.cs [hunk 3]
```diff
@@ -519,17 +508,11 @@ private static bool IsVersionSupported(WorkerLanguageVersionInfo info)
...
-                    case 9:
-                        // We currently do not support Python 3.9 on Windows.
-                        if (PlatformHelper.IsWindows) { return false; }
-                        else return true;
-                    case 8:
-                    case 7:
-                    case 6: return true;
-                    default: return false;
-                }
+                    9 or 8 or 7 or 6 => true,
```
Comment (Hazhzeng) [inline src/Azure.Functions.Cli/Helpers/PythonHelpers.cs:513]: Seems like the pipeline is complaining the new syntax. Let's use the old fashion one.

**Comment by Hazhzeng** [comment]
approved

**Review by vrdmr — APPROVED** [review]

**Pull request merged**